eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -422,8 +422,8 @@
 <figure {{ block( 'field_attributes' ) }}>
     {% set imageAlias = ez_image_alias( field, versionInfo, parameters.alias|default( 'original' ) ) %}
     {% set src = imageAlias ? asset( imageAlias.uri ) : "//:0" %}
-    {% set width = parameters.width is defined ? parameters.width : imageAlias.width %}
-    {% set height = parameters.height is defined ? parameters.height : imageAlias.height %}
+    {% set width = parameters.width is defined ? parameters.width : (imageAlias ? imageAlias.width : '') %}
+    {% set height = parameters.height is defined ? parameters.height : (imageAlias ? imageAlias.height : '') %}
```

Comment:
This change seems a bit out of scope to me. Maybe a distinct commit ? There are active discussions about width/height handling, afair.

Reply:
Indeed this is bug fix. I will extract it to the separate PR targeted to 6.7. Basically in case of failure `\eZ\Publish\Core\MVC\Symfony\Templating\Twig\Extension\ImageExtension::getImageVariation` returns `null` which in consequense ends up    "Impossible to access an attribute ("width") on a null variable." error.  Ref: https://github.com/adamwojs/ezpublish-kernel/blob/48cc181c32ba042e1e24e90250dba2e0c2016fce/eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/ImageExtension.php#L57